web/themes/new_weather_theme/templates/block/block--weathergov-current-conditions.html.twig

```diff
@@ -1,40 +1,36 @@
-<h2 class="usa-sr-only">Current</h2>
-
 {# Widgets and stuff. This is presented as a row of columns. #}
-<div class="grid-container padding-x-0 tablet:padding-x-2">
-  <div class="grid-row grid-col-8 grid-offset-2">
-    <h3 class="usa-sr-only">Current conditions</h3>
-  </div>
-
-  <div class="grid-container bg-white padding-y-2 shadow-1 grid-col-12 tablet-lg:grid-offset-2 tablet-lg:grid-col-8">
-    <div class="wx-current-conditions grid-row grid-gap">
-      {% if content.error %}
-        {% set message = "There was an error loading the current conditions." | t %}
-        {% include '@new_weather_theme/partials/uswds-alert.html.twig' with { 'level': "error", body: message } %}
-      {% else %}
+<h3 class="wx-visual-h2 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-3 margin-bottom-2">
```

Comment:
Can change `margin-top-3` to `margin-top-0` here because the padding on the tab wrapper is duplicating the 3 units of space below the tabs.